custom/src/main/java/co/elastic/otel/ElasticAutoConfigurationCustomizerProvider.java

```diff
@@ -35,29 +35,63 @@ public class ElasticAutoConfigurationCustomizerProvider
   private static final String RUNTIME_EXPERIMENTAL_TELEMETRY =
       "otel.instrumentation.runtime-telemetry.emit-experimental-telemetry";
 
+  // must match value in io.opentelemetry.contrib.stacktrace.StackTraceAutoConfig
+  private static final String STACKTRACE_OTEL_FILTER =
+      "otel.java.experimental.span-stacktrace.filter";
```

Comment:
These are again faulty config option names: due to containing a `-`-character, it is impossible to provide those via environment variables. We should fix this in the upstream.